.travis.yml

```diff
@@ -4,16 +4,11 @@ language: python
 
 python:
   - 2.7
-  - 3.6
 
 env:
   - TOXENV=django111
-  - TOXENV=django20
 
 matrix:
```

Comment:
@OmarIthawi I'm far from an expert in tox, but shouldn't be the opposite? I think we should include python 2.7 and exclude 3.6?

Reply:
You're right @melvinsoft. It's a bit confusing now. I've simplified the envs so they're easier to reason about.